mirrord/kube/src/api/target_list.rs

```diff
@@ -0,0 +1,53 @@
+use k8s_openapi::api::core::v1::{Namespace, Pod};
+use kube::{api::ListParams, Api, Client};
+use serde_json::json;
+
+use crate::error::Result;
+
+async fn get_kube_namespaced_pods(namespace: &Option<&str>) -> Result<Vec<(String, Vec<String>)>> {
```

Comment:
use our wrapper for KubeApi as it holds more important stuff (SSL, namespacing dealing, etc)